\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage{longtable}
\usepackage{graphicx}
\usepackage[a4paper,left=20mm,right=20mm, top=3cm, bottom=3cm]{geometry}
\usepackage{lscape}
\usepackage{hyperref}

\graphicspath{ {figures/} }

\title{African Genome Diversity Reference Panel and Chip Design}
\author{Tommy Carstensen, Martin Pollard, Deepti Gurdasani, Cristina Pomilla}
\date{December 2014}

\begin{document}

\bibliographystyle{plain}
\bibliography{references/references}

\maketitle
\centerline{\includegraphics[width=60mm]{sang_logo_large}}

\include{sections/1rationale}
\include{sections/2datasetsummary}
\include{sections/3curation}
\include{sections/4merging}
%\include{sections/5evalcaller}
\include{sections/5tagSNPselection}

We don't filter based on the percentage of mapped reads. Instead we filter based on heterozygosity. The two parameters are related.

“Version 4 has multiple improvements: support for multi-allelic markers” – BEAGLE4 site

An improved method for calling genotypes and phasing low-coverage data when one also has chip data is to phase the chip data to make a haplotype scaffold and then phase the sequence data onto the scaffold. Both steps are carried out with SHAPEIT2.

\end{document}
