\documentclass{article}
\usepackage[utf8]{inputenc}
%\usepackage{longtable} % for tables
\usepackage{graphicx} % for tables
\usepackage[a4paper,left=20mm,right=20mm, top=3cm, bottom=3cm]{geometry}
\usepackage{lscape}
\usepackage{hyperref}
\usepackage{subcaption}
\usepackage{caption}
\usepackage[table,xcdraw]{xcolor} % for tables
\usepackage{multirow} % for tables

\graphicspath{ {figures/} }

\title{African Diversity Reference Panel and H3A Chip Design}
%\author{Deepti Gurdasani \\ \textttt{dg11@sanger.ac.uk} \and Gerrit Botha \\ \textttt{gerrit.botha@uct.ac.za} \and Martin Pollard \\ \textttt{mp15@sanger.ac.uk} \and Tommy Carstensen \\ \textttt{tc9@sanger.ac.uk} \and Cristina Pomilla \\ \textttt{cp8@sanger.ac.uk} \and Ayton Meintjes \\ \textttt{ayton@cbio.uct.ac.za}}
\date{June 10th, 2015}

\begin{document}

\maketitle
%\centerline{\includegraphics[width=80mm]{sang_logo_large}}

\include{sections/1rationale}
\include{sections/2datasetsummary}
\include{sections/6additional}
\include{sections/3curation}
\include{sections/4tagSNPselection}
\include{sections/5validation}
%\include{sections/7collaborators}

\bibliographystyle{unsrt}%Used BibTeX style is unsrt
\bibliography{references/references}

%We don't filter based on the percentage of mapped reads. Instead we filter based on heterozygosity. The two parameters are related.

%“Version 4 has multiple improvements: support for multi-allelic markers” – BEAGLE4 site

%An improved method for calling genotypes and phasing low-coverage data when one also has chip data is to phase the chip data to make a haplotype scaffold and then phase the sequence data onto the scaffold. Both steps are carried out with SHAPEIT2.

\end{document}
