@article{Menelaou2013,
note = {MVNCall},
author = {Menelaou, Androniki and Marchini, Jonathan}, 
title = {Genotype calling and phasing using next-generation sequencing reads and a haplotype scaffold},
volume = {29}, 
number = {1}, 
pages = {84-91}, 
year = {2013}, 
doi = {10.1093/bioinformatics/bts632}, 
abstract ={Motivation: Given the current costs of next-generation sequencing, large studies carry out low-coverage sequencing followed by application of methods that leverage linkage disequilibrium to infer genotypes. We propose a novel method that assumes study samples are sequenced at low coverage and genotyped on a genome-wide microarray, as in the 1000 Genomes Project (1KGP). We assume polymorphic sites have been detected from the sequencing data and that genotype likelihoods are available at these sites. We also assume that the microarray genotypes have been phased to construct a haplotype scaffold. We then phase each polymorphic site using an MCMC algorithm that iteratively updates the unobserved alleles based on the genotype likelihoods at that site and local haplotype information. We use a multivariate normal model to capture both allele frequency and linkage disequilibrium information around each site. When sequencing data are available from trios, Mendelian transmission constraints are easily accommodated into the updates. The method is highly parallelizable, as it analyses one position at a time.Results: We illustrate the performance of the method compared with other methods using data from Phase 1 of the 1KGP in terms of genotype accuracy, phasing accuracy and downstream imputation performance. We show that the haplotype panel we infer in African samples, which was based on a trio-phased scaffold, increases downstream imputation accuracy for rare variants (R2 increases by >0.05 for minor allele frequency <1%), and this will translate into a boost in power to detect associations. These results highlight the value of incorporating microarray genotypes when calling variants from next-generation sequence data.Availability: The method (called MVNcall) is implemented in a C++ program and is available from http://www.stats.ox.ac.uk/âˆ¼marchini/#software.Contact: marchini@stats.ox.ac.ukSupplementary information: Supplementary data are available at Bioinformatics online.}, 
URL = {http://bioinformatics.oxfordjournals.org/content/29/1/84.abstract}, 
eprint = {http://bioinformatics.oxfordjournals.org/content/29/1/84.full.pdf+html}, 
journal = {Bioinformatics} 
}


@article{Xu2007,
note = {TAGster},
author = {Xu, Zongli and Kaplan, Norman L. and Taylor, Jack A.}, 
title = {TAGster: efficient selection of LD tag SNPs in single or multiple populations},
volume = {23}, 
number = {23}, 
pages = {3254-3255}, 
year = {2007}, 
doi = {10.1093/bioinformatics/btm426}, 
abstract ={Summary: Genetic association studies increasingly rely on the use of linkage disequilibrium (LD) tag SNPs to reduce genotyping costs. We developed a software package TAGster to select, evaluate and visualize LD tag SNPs both for single and multiple populations. We implement several strategies to improve the efficiency of current LD tag SNP selection algorithms: (1) we modify the tag SNP selection procedure of Carlson et al. to improve selection efficiency and further generalize it to multiple populations. (2) We propose a redundant SNP elimination step to speed up the exhaustive tag SNP search algorithm proposed by Qin et al. (3) We present an additional multiple population tag SNP selection algorithm based on the framework of Howie et al., but using our modified exhaustive search procedure. We evaluate these methods using resequenced candidate gene data from the Environmental Genome Project and show improvements in both computational and tagging efficiency.Availability: The software Package TAGster is freely available at http://www.niehs.nih.gov/research/resources/software/tagster/Contact: taylor@niehs.nih.govSupplementary information: Additional information, including a tutorial, detailed algorithm and detailed evaluation results, is also available from TAGster web site (see above).}, 
URL = {http://bioinformatics.oxfordjournals.org/content/23/23/3254.abstract}, 
eprint = {http://bioinformatics.oxfordjournals.org/content/23/23/3254.full.pdf+html}, 
journal = {Bioinformatics} 
}