\section{Generation of haplotypes from raw reads}
Given the need to include large sample sizes from diverse populations for such a resource for development of an African chip, curation of data will involve generation of homogenised data from different datasets, including publicly available data as outlined in Table 1. Given the heterogeneity in sequencing and coverage among datasets (Table 1), these will need to be processed in subsets, and then merged to generate a complete panel of all variants. 

Publicly available curated data, such as reads (bam files) and variant calls (vcf files) from the \href{http://www.1000genomes.org}{1000 Genomes project} and HGDP
%and the \href{http://www.simonsfoundation.org/}{Simonâ€™s Foundation}
will be used as such. For datasets that are sequenced in house, consistent methods will be used for curation.

\input{sections/curation/preprocessing}

\input{sections/curation/QCprecalling}

\input{sections/curation/caller_eval}

\input{sections/curation/calling}

\input{sections/curation/filtering}

\input{sections/curation/merging}

\input{sections/curation/refinement}

\input{sections/curation/QCpostrefinement}