\section{Curation}
Given the need to include large sample sizes from diverse populations for such a resource for development of an African chip, curation of data will involve generation of homogenised data from different datasets, including publicly available data as outlined in Table 1. Given the heterogeneity in sequencing and coverage among datasets (Table 1), these will need to be processed in subsets, and then merged to generate a complete panel of all variants. 

Publicly available curated data, such as from the \href{http://www.1000genomes.org}{1000 Genomes project}, HGDP and the \href{http://www.simonsfoundation.org/}{Simonâ€™s Foundation} will be used as such. For datasets that are sequenced in house, consistent methods will be used for curation.

Following generation of raw reads, duplicate reads are marked, and mapping will be carried out using BWA/BWA-mem to the human reference genome that is current at the time (GRCh37/GRCh38). Lanes will be merged into samples, and sample level bam improvement will be carried out using GATKv2.8+. This process will consist of of re-alignment of reads around indels in addition to recalibration of base qualities using the GATK BQSR function. One sample from the Genome in a Bottle (GIAB) highly curated set will be included for validation of the data processing pipeline (NA12878)1 (http://genomeinabottle.org) in each subset. PCR-free reads will be used for these validation samples, to avoid PCR artefacts. The validation sample will be downsampled to the coverage of the dataset being called, and processed through the same pipeline, to provide a comparator against a validated highly curated set of variants for this sample. The GIAB sample (NA12878) represents a sample from a 12 person pedigree that has gone through extensive curation and validation of variants. 1The accuracy of called data from this sample will provide a guide to the accuracy of the workflow applied. After applying standard QC filtering to bams, variant calling will be carried out using several algorithms (Samtools, Haplotype caller and Unified Genotyper). Calling of samples will be carried out with the GIAB genome sample, in order to assess the sensitivity and specificity of any calling pipeline for SNPs and structural variants. The pipeline with the greatest area under the ROC sensitivity/1-specificity curve will be used for calling variants (these callers may be different for SNPs and structural variants). Calling and filtering of SNPs and indels will be carried out separately using the chosen algorithm(s), with filtering thresholds chosen based on the sensitivity and specificity on the platinum genomes sample. 

Datasets generated with unique Illumina chemistry, and those with different coverages will need to be called in separate subsets, if a multi-sample calling algorithm such as Unified Genotyper or Samtools is used. If Haplotype Caller is used for calling, calling will be carried out per sample, as per best practice recommendations (https://www.broadinstitute.org/gatk/guide/best-practices).